tableaudocumentapi/multilookup_dict.py

```diff
@@ -40,6 +46,14 @@ def __setitem__(self, key, value):
 
         dict.__setitem__(self, key, value)
 
+    def get(self, key, default_value=AttributeError):
+        try:
+            return self[key]
+        except AttributeError:
+            if default_value != AttributeError:
```

Comment:
I don't know how equality checking works with Exceptions deep down, but I think it's safer to do an isinstance check.

`except AttributeError:
    if not isinstance(default_value, AttributeError)`